src/edwards/point.rs

```diff
@@ -29,6 +30,17 @@ impl EcPoint<Scalar> for Point {
     fn is_zero(&self) -> bool {
         self.c.x.is_zero() && !self.c.y.is_zero() && self.c.t.is_zero() && !self.c.z.is_zero()
     }
+    // based on https://tools.ietf.org/html/rfc8032#section-5.2.2
+    fn serialize(&self, _: bool) -> Vec<u8> {
+        let x = &self.c.x;
+        let y = &self.c.y;
+        // negative == odd
+        let x_0 = (((x.sgn0_le()>>1)&0x01)<<7) as u8;
+        let mut enc = y.to_bytes_be();
+        enc[0] = enc[0] | x_0;
+        enc.reverse(); // le
```

Comment:
you can also use `to_bytes_le`

Reply:
This is a method on FpElt, so `to_bytes_le` doesn't exist yet.

Reply:
I've written it now 👍